### DISCUSSION TITLE
Scrolling Sidebar

### DISCUSSION THREAD

**Discussion by kryptonianson** [keiwando/toonsquid#169] [Ideas]
When the timeline is expanded it covers undo/redo. It would be nice if the sidebar would scroll when the timeline is expanded. This will probably be helpful as additional buttons are added as features are added.

**Comment by Cat-Butler** [comment]
Workarounds in the meantime (you are probably well aware of these, but for the benefit of new users reading the forum):
 
ToonSquid supports the Procreate-style two-finger tap to undo, three-finger tap to redo.

You can also drag the top edge of the timeline up and down to reveal more or less of the working screen.
Even on a iPad Mini 5th gen., which has one of the lower screen resolutions for a modern iPad, I can show the undo/redo icons.

Admittedly, you are then limiting the timeline display, possibly to a single layer, but the timeline content does scroll vertically.
If you have an old iPad, you're going to be limited in screen resolution, so this might not work.
